torch/distributed/_shard/sharding_spec/__init__.py

```diff
@@ -6,7 +6,7 @@
     _infer_sharding_spec_from_shards_metadata,
 )
 from .chunk_sharding_spec import (
-    ChunkShardingSpec,
+    ChunkShardingSpec as ChunkShardingSpec,
```

Comment:
For my learning purpose, why is this needed?

Reply:
This is for being able to import ChunkShardingSpec, otherwise public access isn't defined and linters complain, see #80433 for a similar issue.

Reply:
Got it...It's just the way the code looks like is bit weird to me.